docs/SchemeFileFormat.md

```diff
@@ -86,7 +86,7 @@ The supported entries in the outputConfiguration object are as follows:
 | subset           | A dict object with the defined subset elements to display in the output                                         | [subset](#subset) object                                     |
 | fullExplanation  | Choose if to print all counterexamples causing the query result in the output                                   | bool                                                         |
 | excludeIPv6Range | If the policies of the config do not contain any IPv6 addresses, do not include IPv6 range in the query results | bool  [default: True]                                        |
-
+| explain          |A pair of node names to explain the policies affecting their connection or lack of it. When 'ALL' is supplied, all the pairs in the topology are explained.| string [ ns/node1,ns/node2 / ALL ]                           |
```

Comment:
- should mention it is only supported for connectivity query.
- how it can be used for cli query? 
- add examples how this can be used, and example input/output (consider a separate section/page about this feature) .
- what if one of the nodes is an ip-block?

Reply:
Done. Also removed 'ALL' since it is for internal use.

Reply:
so it is not possible to get `explain` for a connectivity between deployment and a certain ip-address? do we want to specify this, or create an issue to consider support this later?

Reply:
maybe you missed this comment from last iteration?

Reply:
explanation is possible for all nodes-pairs in the topology, including ip-blocks. The ip-blocks input format was added to the docs so it will be more clear how to use the feature.